Stream_support/doc/Stream_support/CGAL/IO/io.h

```diff
@@ -289,4 +289,153 @@ The input operator is defined for all classes in the \cgal `Kernel`.
 */
 istream& operator>>(istream& is, Class c);
 
+
+//! \ingroup PkgIOstreams
+//! \brief `read_point_WKT()` fills a `Point` from a  WKT stream. The first line starting with POINT 
+//! in the stream will be used. 
+//! 
+//! \tparam Point can be a `CGAL::Point_2` or `CGAL::Point_3`.
+//! 
+//! \see `CGAL::Point_2`
+//! \see `CGAL::Point_3`
+template<typename Point>
+std::istream&
+read_point_WKT( std::istream& in,
+                Point& point );
+
+//! \ingroup PkgIOstreams
+//! \brief `read_multi_point_WKT()` overwrites the content of a `MultiPoint` 
+//! with the first line starting with MULTIPOINT in the stream.
+//! 
+//! \tparam MultiPoint must be a model of `RandomAccessRange` of `CGAL::Point_2` or `CGAL::Point_3`.
+//! 
+//! \see `CGAL::Point_2`
+//! \see `CGAL::Point_3`
+template<typename MultiPoint>
+std::istream&
+read_multi_point_WKT( std::istream& in,
+                      MultiPoint& mp );
+
+
+//! \ingroup PkgIOstreams
+//! \brief `read_linestring_WKT()` fills a `Linestring` from a WKT stream.
+//! The first line starting with LINESTRING in the stream will be used.
+//!
+//! \tparam Linestring must be a model of `RandomAccessRange` of `CGAL::Point_2`.
```

Comment:
What is a `RandomAccessRange`?

Reply:
https://www.boost.org/doc/libs/1_67_0/libs/range/doc/html/range/concepts/random_access_range.html
It is a range with a RandomAccessIterator

Reply:
I see no mutable capabilities, does it mean it need to be allocated first? How do you actually use it?

Reply:
Well you declare a random access range of Point_2, a std::vector<CGAL::Point_2<K> > by default, and you give it to the fucntion. You don't need to allocate it first, which is rather convenient as the WKT format is a streaming format, so you don't know the size of a linestring before you read it entirely. There are exemples to show the use of the functions.

Reply:
That is suspect. If the range is not resized before its use, then it means that the algorithm does the allocation or the resizing, but the concept of `RandomAccessRange` does not have anything that allows that.

Reply:
It uses push_back(), which I found nowhere in the doc.

Reply:
@maxGimeno Maybe this is what you need : [SequenceContainer](https://en.cppreference.com/w/cpp/named_req/SequenceContainer)

Reply:
I don't need anything, it's just the doc of boost that is not complete enough about WKT usage, so apparently I cannot just forward the concepts, I have to complete it with what they forgot.

Reply:
Here the comment is still valid, how do you add element in the container? Did you look at my suggestion?